go.mod

```diff
@@ -8,6 +8,8 @@ go 1.24.2
 // This retraction is to prevent it from being used and from breaking builds of dependent projects.
 retract v1.2.2
 
+exclude sigs.k8s.io/gateway-api v1.3.0
```

Comment:
Why are we excluding gateway-api here?

Reply:
Without this stanza `go mod` was bumping this to 1.3 which I do not want to include in this PR.

Reply:
gwapi 1.3 is already on main https://github.com/Kong/gateway-operator/blob/main/go.mod#L45